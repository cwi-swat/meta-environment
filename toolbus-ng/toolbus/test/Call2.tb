process P(N : int) is Q(N) . print N

process Q(N : int) is N := mul(N,2) . R(N) . print N

process R(N : int) is N := mul(N,3) . print N

toolbus(P(5))
