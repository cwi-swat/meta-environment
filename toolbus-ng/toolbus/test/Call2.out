30
10
5
